fedbiomed/node/requests/_protocol_manager.py

```diff
@@ -0,0 +1,231 @@
+# This file is originally part of Fed-BioMed
+# SPDX-License-Identifier: Apache-2.0
+
+from threading import Thread
+import asyncio
+import time
+
+from fedbiomed.common.constants import ErrorNumbers
+from fedbiomed.common.exceptions import FedbiomedNodeToNodeError
+from fedbiomed.common.logger import logger
+
+from fedbiomed.node.environ import environ
+from ._overlay import format_incoming_overlay
+from ._pending_requests import PendingRequests
+from ._protocol_handler import ProtocolHandler
+
+from fedbiomed.transport.controller import GrpcController
+
+
+# Maximum number of pending messages in the protocol manager input queue
+MAX_PROTOCOL_MANAGER_QUEUE_SIZE = 1000
+
+# Maximum duration in seconds of overlay message processing task
+OVERLAY_MESSAGE_PROCESS_TIMEOUT = 60
+
+
+class _ProtocolAsyncManager:
```

Comment:
These are just a suggestion. My point is the specify that it is for node to node communication. 
`ProtocolHandler` -> `N2NController`
`ProtocolManager` -> `N2NRouter`

Reply:
Thank you @srcansiz, I pushed:
- `ProtocolHandler` => `NodeToNodeHandler`
- `ProtocolManager` => `NodeToNodeRouter`

... just to avoid that discussion over `N2NRouter` vs `N2nRouter` (and uncomfortable camel case output in both cases ;-) )